### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by ianthomas23**
This fixes a few typos.

### CODE REVIEW

File: .github/workflows/bokeh-ci.yml [hunk 1]
```diff
@@ -51,7 +51,7 @@ jobs:
         run: bash scripts/ci/verify_pip_install_using_wheel.sh
 
       - name: Build conda package
-        run: bash scripts/ci/build_conda_pacakge.sh
+        run: bash scripts/ci/build_conda_package.sh
```
Comment (bryevdv) [inline .github/workflows/bokeh-ci.yml:54]: How was this not breaking CI?

**Review by bryevdv — DISMISSED** [review]

File: .github/workflows/bokeh-ci.yml (hunk 1 above)
Comment (ianthomas23) [reply .github/workflows/bokeh-ci.yml:54]: The script was similarly named so everything worked as planned. I've renamed the script in line with this change.

**Comment by ianthomas23** [comment]
I was keeping this WIP until CI passed. Windows failure looks unrelated and hopefully something transient, so I'll look again tomorrow.

**Comment by ianthomas23** [comment]
Test failures are unrelated so this is now ready for review.

**Review by mattpap — APPROVED** [review]

**Pull request merged**